auth_otp/serializers.py

```
@@ -0,0 +1,375 @@
+        model = User
+        fields = [
+            'id',
+            'username',
+            'email',
+            'first_name',
+            'last_name',
+            'full_name',
+            'user_type',
+            'is_active',
+            'created_at'
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)